src/main/kotlin/icons/CheckovIcons.kt

```diff
@@ -6,5 +6,9 @@ object CheckovIcons {
 
     val ErrorIcon = IconLoader.getIcon("/icons/balloonError.svg", javaClass)
     val TerraformIcon = IconLoader.getIcon("/icons/terraform.svg", javaClass)
-    val SeverityIcon = IconLoader.getIcon("/icons/SeverityError.svg", javaClass)
+    val SeverityUnknown = IconLoader.getIcon("/icons/severity_unknown.svg", javaClass)
+    val SeverityLow = IconLoader.getIcon("/icons/severity_low.svg", javaClass)
+    val SeverityMedium = IconLoader.getIcon("/icons/severity_medium.svg", javaClass)
+    val SeverityHigh = IconLoader.getIcon("/icons/severity_high.svg", javaClass)
+    val SeverityCritical = IconLoader.getIcon("/icons/severity_critical.svg", javaClass)
```

Comment:
the icons appear not unified in the github preview (size, ratio, background color) is that effecting in the jetbrains view?

Reply:
in JetBrains they look fine, as seen in the video :)